modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/Skeleton.js

```diff
@@ -9,7 +9,7 @@
  * distribution rights of the Software.
  */
 
-import SupportOverview from '../../../components/SupportOverview';
+import SupportOverview from './SupportOverview/SupportOverview';
```

Comment:
```suggestion
import SupportOverview from './SupportOverview';
```

Notei que SupportOverview não está dentro de uma pasta components da Overview, não deveríamos criar uma? Faz sentido?

Reply:
Perfeito Clarice. Vou fazer a alteração.